syntax = "proto3";

package showtime;

service Showtime {
  rpc GetSchedule (Empty) returns (Schedule) {}
  rpc GetMoviesByDate (Date) returns (MovieSchedule) {}
}

message Empty {}

message Date {
  string date = 1;
}

message MovieSchedule {
  string date = 1;
  repeated string movie_ids = 2;
}

message Schedule {
  repeated MovieSchedule schedules = 1;
}